deprecation.rst

```diff
@@ -17,3 +17,18 @@ no version bumps are expected.
 
 The current list of deprecated EAPIs is stored as ``eapis-deprecated``
 in ``metadata/layout.conf``.
+
+
+.. index:: EAPI; banned
+
+Banned EAPIs
+------------
+:PG: 1002
+:Source: Council
+:Reference: https://gitweb.gentoo.org/repo/gentoo.git/tree/metadata/layout.conf
+:Reported: by pkgcheck and repoman
+
+Banned EAPIs must not be used by any ebuilds in the repository.
```

Comment:
But they are?

Reply:
Hmm, indeed I have made my assumptions based on EAPIs that are listed as banned here.